.github/workflows/copilot-setup-steps.yml

```
@@ -1 +1,4 @@
 #
+
+permissions:
```

actionlint (1.7.12)

[error] "jobs" section is missing in workflow

(syntax-check)